src/app/issuer/components/issuer-staff/issuer-staff.component.ts

```diff
@@ -84,6 +84,9 @@ export class IssuerStaffComponent extends BaseAuthenticatedRoutableComponent imp
 
 	dialogRef: BrnDialogRef<any> = null;
 
+	editMembers = "Mitglieder bearbeiten"
+	members = "Mitglieder"
```

Comment:
As mentioned before, this should probably better start off with `this.translate.instant`